doc/source/installation/apt_repo.rst

```diff
@@ -146,6 +145,95 @@ This will install all the packages from the bundle. Another option is to downloa
 
   When installing packages manually like this, you'll need to make sure to resolve all the dependencies and install missing packages yourself. Following packages will need to be installed before you can manually install Percona Server: ``mysql-common``, ``libjemalloc1``, ``libaio1`` and ``libmecab2``
 
+.. rubric:: AppArmor settings
+
+AppArmor is a kernel-integrated system which controls how applications access the file system by creating application profiles. If the installation of MySQL adds an AppArmor profile, the profile is in the following locations:
+
+* /etc/apparmor.d/usr.sbin.mysqld
+* /etc/apparmor.d/local/usr.sbin.mysqld
+
+The ``local`` version is empty except for comments. Add any changes specific for the server to the ``local`` file. 
```

Comment:
The ``local`` version contains only comments.

Reply:
fixed